sdk/appcenter-storage/src/test/java/com/microsoft/appcenter/storage/StorageTest.java

```diff
@@ -267,12 +388,43 @@ public void buildAppCenterGetDbTokenBodyPayload() {
         // These constructors must be called even though these classes are not going to be instantiated.
         TokenExchange te = new TokenExchange();
         CosmosDb cdb = new CosmosDb();
+        Utils utils = new Utils();
+        Utils.handleApiCallFailure(new HttpException(200, "b"));
+    }
+
+    @Test
+    public void documentDeserialization() {
+        Document<TestDocument> d =
+                Utils.parseDocument(cosmosDbDocumentResponsePayload, TestDocument.class);
+        assertEquals(DOCUMENT_ID, d.getId());
+        assertEquals(PARTITION, d.getPartition());
+        assertEquals(TEST_FIELD_VALUE, d.getDocument().test);
+    }
+
+    @Test
+    public void documentSerialization() {
+        assertEquals(
```

Comment:
in the next method you improved formatting by using local variable, this one would really benefit local variables too